presto-main/src/main/java/com/facebook/presto/execution/TaskExecutor.java

```diff
@@ -855,6 +899,16 @@ public long getRunningTasksLevel4()
     }
 
     @Managed
+    public long getLongestSplitTimeNanos()
+    {
+        Iterator<SplitProfile> iterator = splitProfiler.iterator();
+        if (iterator.hasNext()) {
+            return System.nanoTime() - iterator.next().getStartTime();
```

Comment:
Let's rename this method to millis, and divide this number by 1_000_000. It's too large otherwise for human consumption otherwise. (But you should still use nanoTime.)